corehq/apps/registration/views.py

```diff
@@ -117,7 +122,22 @@ def _create_new_account(self, reg_form):
             web_user = WebUser.get_by_username(new_user.username)
             web_user.phone_numbers.append(reg_form.cleaned_data['phone_number'])
             web_user.save()
-        track_workflow(new_user.email, "Requested new account")
+
+        is_mobile = reg_form.cleaned_data.get('is_mobile')
+        email = reg_form.cleaned_data['email']
```

Comment:
`email` and `new_user.email` are the same, right? Would be more readable to use one.